packages/fast-usdc/package.json

```diff
@@ -1,34 +1,64 @@
 {
-  "name": "fast-usdc",
-  "private": true,
-  "version": "0.1.1-u18.2",
-  "description": "Create an Agoric Javascript smart contract application",
+  "name": "@agoric/fast-usdc",
+  "version": "0.1.0",
+  "description": "CLI and library for Fast USDC product",
   "type": "module",
   "files": [
     "contract",
-    "src"
+    "src",
+    "tools"
   ],
   "bin": {
-    "fast-usdc": "./src/cli.js"
+    "fast-usdc": "./src/cli/bin.js"
   },
   "scripts": {
     "build": "exit 0",
     "test": "ava",
-    "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
+    "test:c8": "c8 --all $C8_OPTIONS ava",
     "test:xs": "exit 0",
     "lint-fix": "yarn lint:eslint --fix",
     "lint": "run-s --continue-on-error lint:*",
     "lint:types": "tsc",
     "lint:eslint": "eslint ."
   },
   "devDependencies": {
+    "@agoric/swingset-liveslots": "^0.10.2",
+    "@agoric/vats": "^0.15.1",
+    "@agoric/zone": "^0.2.2",
+    "@fast-check/ava": "^2.0.1",
     "ava": "^5.3.0",
-    "c8": "^9.1.0",
-    "ts-blank-space": "^0.4.1"
+    "c8": "^10.1.2",
+    "execa": "9.1.0",
+    "ts-blank-space": "^0.4.4"
   },
   "dependencies": {
-    "agoric": "^0.22.0-u18.2",
-    "commander": "^12.1.0"
+    "@agoric/client-utils": "^0.1.0",
+    "@agoric/cosmic-proto": "^0.4.0",
+    "@agoric/ertp": "^0.16.2",
+    "@agoric/internal": "^0.3.2",
+    "@agoric/notifier": "^0.6.2",
+    "@agoric/orchestration": "^0.1.0",
+    "@agoric/store": "^0.9.2",
+    "@agoric/vat-data": "^0.5.2",
+    "@agoric/vow": "^0.1.0",
+    "@agoric/zoe": "^0.26.2",
```

Comment:
@mhofman I believe we need to match the agoric deps versions with the rest of the packages (with u18 pre-ids). Will need to do it manually. Kindly confirm